### DISCUSSION TITLE
Is it possible to pass quarkus runtime properties through environment variables?

### DISCUSSION THREAD

**Discussion by djotanov** [keycloak/keycloak#12305] [Archive]
Hi,

in my keycloak instance I am using UserStoreFederationProvider, similar to the one from quickstart examples: https://github.com/keycloak/keycloak-quickstarts/tree/main/user-storage-jpa
I would like to avoid hardcoding database credentials in quarkus properties, and passing them through JAVA_OPTS_APPEND environment variable isn't ideal either.
Is it possible to pass, let's say quarkus.datasource.user-store.password, as environment variable?

**Comment by danielgratzl** [comment]
@djotanov Did you find a solution yet?
